source/llvm/constants.d

```diff
@@ -272,6 +272,53 @@ static if (LLVM_Version >= asVersion(3, 5, 0))
 }
 
 static if (LLVM_Version >= asVersion(3, 4, 0))
```

Comment:
I think this should be changed to `static if (LLVM_Version >= asVersion(7, 0, 0))`.

Reply:
ty, is fixed